documentation/adding_and_updating_endpoints.md

```diff
@@ -0,0 +1,83 @@
+# Adding And Updating New Phoenix Endpoints
+
+***
+
+
```

Comment:
Yeah, sounds good. I might reference a link to it in this endpoints doc to the wiki though.